ot-node.js

```diff
@@ -109,26 +109,29 @@ class OTNode {
         this.logger.info('Event emitter initialized');
     }
 
-    async initializeControllers() {
+    async initializeRouters() {
         try {
-            this.logger.info('Initializing http api router');
+            this.logger.info('Initializing http api and rpc router');
             const httpApiRouter = this.container.resolve('httpApiRouter');
-            await httpApiRouter.initialize();
-        } catch (e) {
-            this.logger.error(
-                `Http api router initialization failed. Error message: ${e.message}, ${e.stackTrace}`,
-            );
-            this.stop(1);
-        }
-
-        try {
-            this.logger.info('Initializing rpc router');
             const rpcRouter = this.container.resolve('rpcRouter');
-            await rpcRouter.initialize();
+
+            await Promise.all([
+                httpApiRouter.initialize().catch((err) => {
+                    this.logger.error(
+                        `Http api router initialization failed. Error message: ${err.message}, ${err.stackTrace}`,
+                    );
+                    this.stop(1);
+                }),
+                rpcRouter.initialize().catch((err) => {
```

Comment:
there is no need for these catch blocks since there is the catch at line 133

Reply:
I wanted to have different log statement for each router, so it's easier to debug later. Global try/catch is added if for any reason we are not able to resolve routers objects from dependency container.